assets/stylesheets/sass/main/main.scss

```diff
@@ -230,6 +234,35 @@ figure {
   }
 }
 
+.no-cssgradients {
+
+  .hero {
+    background-image: url("../../../images/new-hero-ie.jpg");
+  }
+}
+
+.no-svg {
+
+  .social-buttons {
+
+    div {
+
+      &.facebook {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Selector should have depth of applicability no greater than 2, but was 3